libs/titanc/src/compiler/ast.rs

```diff
@@ -5,6 +5,17 @@ use crate::compiler::debug::Span;
 
 #[derive(Debug)]
 pub struct Scope {
-  pub text: String, // this is only here until we add change the grammar
-  pub span: Span
-}
\ No newline at end of file
+	pub statements: Vec<Statement>,
+	pub span: Span,
+}
+
+#[derive(Debug, Clone)]
+pub struct Statement {
+	pub kind: StatementKind,
+	pub span: Span,
+}
+
+#[derive(Debug, Clone)]
+pub enum StatementKind {
+	Let { identifier: String, value: String },
```

Comment:
Lets add a field for type maybe something like 
```suggestion
	Let { identifier: String, value: String, type_:  TypeKind },
```

which means we probably need another enum for TypeKind
```rust
#[derive(Debug)]
pub enum TypeKind {
	Number,
	String,
	Bool,
}
```